.github/workflows/ci.yaml

```diff
@@ -36,6 +37,25 @@ jobs:
             gromacs-version: "2018.6"
           - os: macOS-latest
             gromacs-version: "2020.6"
+          - os: macOS-latest
```

Comment:
The above excludes can be removed now because these gromacs versions are not in the matrix anymore.